rplugin/python3/denite/source/file_rec.py

```diff
@@ -25,18 +25,19 @@ def __init__(self, vim):
         self.__cache = {}
 
     def on_init(self, context):
-        self.__proc = None
-        directory = context['args'][0] if len(
-            context['args']) > 0 else context['path']
-        context['__directory'] = self.vim.call('expand', directory)
+        context['__proc'] = None
+        try:
+            context['__directory'] = context['args'][0]
+        except IndexError:
+            context['__directory'] = self.vim.call('getcwd')
```

Comment:
Please don't change the current behavior.
Where is `self.vim.call('expand', directory)`?
It is needed.